presto-docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -50,6 +50,20 @@ property, which can be set to ``UNNECESSARY`` (the default),
 ``UP``, ``DOWN``, ``CEILING``, ``FLOOR``, ``HALF_UP``, ``HALF_DOWN``, or ``HALF_EVEN``
 (see `RoundingMode <https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/math/RoundingMode.html#enum.constant.summary>`_).
 
+Array Type Handling
+-------------------
+
+The PostgreSQL array implementation does not support fixed dimensions whereas PrestoSql
+support only arrays with fixed dimensions.
+The way PostgreSQL connector handle array columns is set by using the ``postgresql.array-mapping`` configuration property
```

Comment:
You can configure how the PostgreSQL connector handles arrays with the ``postgresql.array-mapping`` configuration property in your catalog file or the ``array_mapping`` session property.